CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-# 2.48.2 (2021-09-01)
+# 2.49.0 (2021-09-02)
+
+- Added additional domain to the auth domain whitelist. [#203](https://github.com/blackbaud/auth-client/pull/203) (Thanks @Blackbaud-AlexisCoulter)
```

Comment:
The linked issue should still be #202 since that's where the changes to the whitelist were made.